viewer/src/components/items/Table.js

```diff
@@ -1,28 +1,48 @@
 import Loading from "../Loading";
-import React from "react";
+import React, { useState } from "react";
 import { MaterialReactTable } from 'material-react-table';
```

Comment:
Do we want to provide a way for people to enable [aggregation & grouping](https://www.material-react-table.com/docs/guides/aggregation-and-grouping) in this pr while we're working on the tables?

Reply:
I could just turn it on if that is as easy as it needs to be.